cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/release_plan_admission-application-service.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlanAdmission
+metadata:
+  name: rhtap-application-service-rpa
+  namespace: rh-managed-rhtap-ser-tenant
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
+spec:
+  applications:
+    - application-service
+  data:
+    images:
+      addGitShaTag: true
+      defaultTag: latest
+    infra-deployment-update-script: |
+      sed -i -e 's|\(https://github.com/redhat-appstudio/application-service/.*?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/has/base/kustomization.yaml
+      sed -i -e 's|\(https://github.com/redhat-appstudio/application-service/.*?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/has/production/kustomization.yaml
+      sed -i -e 's|\(https://github.com/redhat-appstudio/application-service/config/prometheus/?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/has/production/kustomization.yaml
+      sed -i -e 's|\(https://github.com/redhat-appstudio/application-service/config/prometheus/?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/has/staging/kustomization.yaml
+      sed -i -e 's|\(https://github.com/redhat-appstudio/application-service/config/monitoring/?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/monitoring/grafana/base/dashboards/has/kustomization.yaml    
+    mapping:
+      components:
+        - name: application-service
+          repository: "quay.io/redhat-appstudio/application-service"
+    pyxis:
+      secret: pyxis-staging-secret
```

```suggestion
      secret: pyxis-production-secret
```

where did you see this value?

---

From the onboarding doc: https://docs.google.com/document/d/1nCHMYxyJQW_fAH8z97uk16IvYTQaodbUn87fMGgw8YA/edit#heading=h.ivxgstl1y3e7

---

thanks, I fixed the doc. It should be `production`

---

Ah, good to know. I've applied the change to the file